src/SFGUI/ResourceManager.cpp

```diff
@@ -52,51 +52,24 @@ std::shared_ptr<const sf::Font> ResourceManager::GetFont( const std::string& pat
 	auto loader = GetMatchingLoader( path );
 
 	if( !loader ) {
-		std::shared_ptr<const sf::Font> font;
-
-		if( m_use_default_font ) {
-			auto font_iter = m_fonts.find( path );
-
-			if( font_iter == m_fonts.end() ) {
-#if defined( SFGUI_INCLUDE_FONT )
-				font = std::make_shared<sf::Font>( LoadDejaVuSansFont() );
-#else
+		std::shared_ptr<const sf::Font> font = GetFont( "Default" );
 #if defined( SFGUI_DEBUG )
-				std::cerr << "SFGUI warning: No default font available. (SFGUI_INCLUDE_FONT = FALSE)\n";
-#endif
-				font = std::make_shared<sf::Font>();
+		std::cerr << "SFGUI warning: Couldn't find a loader for the font \"" << path << "\"."
```

Comment:
Combine this into a single-line cerr. Remove the check for the font. Loading the default font will print the message itself.